pyroomacoustics/room.py

```diff
@@ -98,6 +98,37 @@
     Until recently, rooms would take an ``absorption`` parameter that was
     actually **not** the energy absorption we use now.  The ``absorption``
     parameter is now deprecated and will be removed in the future.
+    
+    
+Randomized Image Method
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In highly symmetric shoebox rooms, the regularity of image sources’ positions 
+leads to a monotonic convergence in the time arrival of far-field image pairs.
+This causessweeping echoes. The randomized image method adds a small random 
+displacement to the image source positions, so that they are no longer 
+time-aligned, thus reducing sweeping echoes considerably.
+
+To use the randomized method, set the flag ``use_rand_ism`` to True while creating
+a room. Additionally, the maximum displacement of the image sources can be 
+chosen by setting the parameter ``max_rand_disp``. The default value is 8cm.
```

Comment:
You could reference the example here, e.g., For a full example see `examples/randomized_image_method.py`.